changelog.d/gh-5979.fixed

```diff
@@ -0,0 +1 @@
+Releases were broken, as evidenced by our nightly homebrew tests. This fixes those linker errors so we can start releases.
```

Comment:
"This" sounds like it's referring to the commit or PR, but users reading our changelog won't know that.

Reply:
Besides, if this appears in the changelog, our users would be led to believe that the previous release was faulty. That's not really true, is it?

Reply:
Yeah. i'm in favor of excluding this from changelog